packages/ses/src/compartment-shim.js

```diff
@@ -109,6 +176,51 @@ export class Compartment {
     });
```

Comment:
Line 154 talks about `x` as an option, but it is actually a mandatory positional parameter on like 156.

On line 156, why is it named `x`?

Reply:
In this case, this is where I found Chesterton’s Fence. I would be elated to rename this `source` or `program` and fix the comment.